client/ayon_kitsu/hooks/pre_status_cahnge.py

```diff
@@ -0,0 +1,101 @@
+from ayon_applications import PreLaunchHook
+from ayon_kitsu.addon import is_kitsu_enabled_in_settings
+
+import os
+import gazu
+import re
+
+
+
+class PreStatusChange(PostLaunchHook):
```

Comment:
And can you add docstring explaining what it does.

Reply:
yes i can name it. 

`And can you add docstring explaining what it does.`
this i dont understand

Reply:
Under the class definition add a docstring, which is a few lines explaining how it works:

```suggestion
class PreKitsuStatusChange(PostLaunchHook):
    """On launch of an application change task statuses on Kitsu.

    When launching an application this launch hook may change the
    status on Kitsu with the relevant task id. 

    If "pause other tasks" is enabled this will also change the status of 
    all other tasks to the paused task status. This can be useful for
    X, Y, Z.
    """
```
etc.